test/TestProjects/MgmtPropertyBag/Generated/Models/FooCollectionGetAllOptions.cs

```diff
@@ -18,21 +18,16 @@ public FooCollectionGetAllOptions()
         }
 
         /// <summary> The filter to apply on the operation. </summary>
-        public string Filter { get; set; } = null;
-
+        public string Filter { get; set; }
         /// <summary> The Integer to use. </summary>
-        public int? Top { get; set; } = 10;
```

Comment:
we are losing the default value here, how are we accomodating for this?

Reply:
well this is also an inconsistency just like the null value in arrays.
When we do not have the option bag, we will always have this kind of method:
```csharp
public Response DoSomething(int? top = null)
```
therefore this default value also changes something

Reply:
hmm, shouldn't have the default in both the method and property bag?

Reply:
Sorry I just noticed that this default value is provided by `x-ms-client-default` therefore we need to keep this. I will investigate this to see if without the property bag, it is still there. If it is still there, I will keep it here.

Reply:
I fixed this issue - all the `null` value initializations are removed because they are just redundant.
The `10` is kept as before